docs/wfforward.md

```diff
@@ -90,20 +92,20 @@ def wf1(N, ngens):
     Constant-sized WF model with no mutation, no recombination, no selection,
     and no simplification.
     """
-    nodes, edges = msprime.NodeTable(), msprime.EdgeTable()
+    tc = msprime.TableCollection(1.0)
     
     # Add 2N nodes at time = 0.
     # These nodes represent the 
     # initial list of parental
     # gametes
     for i in range(2*N):
-        nodes.add_row(time=0, flags=msprime.NODE_IS_SAMPLE)
+        tc.nodes.add_row(time=0, flags=msprime.NODE_IS_SAMPLE)
     
-    next_offspring_index = len(nodes)
+    next_offspring_index = len(tc.nodes)
     first_parental_index = 0
     for gen in range(1,ngens+1):
-        assert next_offspring_index == len(nodes)
-        assert first_parental_index == len(nodes) - 2*N
```

Comment:
Making ``assert`` look like a function isn't terribly idiomatic I think, but it's a personal style choice (so up to you).